docs/pages/blog/mui-x-v5.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Introducing MUI X v5.0
+description: We are excited to introduce MUI X v5.0.0!
+date: 2021-11-12T00:00:00.000Z
+authors:
+  ['oliviertassinari', 'm4theushw', 'flaviendelangle', 'DanailH', 'alexfauquette']
+card: true
+---
+
+We are excited to introduce [MUI X v5.0.0](https://github.com/mui-org/material-ui-x/releases/tag/v5.0.0)!
+
+<img src="/static/blog/mui-x-v5/card.png" alt="" style="width: 100%; margin-bottom: 16px;" />
```

Comment:
@mui-org/x v5.0.0 is broke. Should we change to v5.0.1 or announce as v5.0.0 but users will actually install v5.0.1?

Reply:
I think it's still valid to leave it as it is. We will have a patch so it will be installed automatically.